### DISCUSSION TITLE
GPT4All

### DISCUSSION THREAD

**Discussion by BennettFourr** [oobabooga/text-generation-webui#1724] [General]
I use this and GPT-4-All(https://github.com/nomic-ai/gpt4all) and can't see any big advantage to this one other than compatibility. Are there any other advantages to this?

**Comment by Felixqian4160** [comment]
can this model run in the text-generation-webui？？

**Comment by BetaDoggo** [comment]
The point of this ui is that it runs everything. The GPT4all ui only supports gpt4all models so it's extremely limited. I believe the gpt4all ui also doesn't support gpu compute but I might be wrong about that.
